perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -1,10 +1,22 @@
 #!/bin/bash
 # It will take approx 80 minutes to run the script.
-set -e
 sudo apt-get update
 
 echo Installing git
 sudo apt-get install git
+
+# Running test only for when PR contains execute-perf-test label
```

Comment:
If this block doesn't not have any dependency over git. We can move this to the top.

Reply:
fixed